src/V3Error.h

```diff
@@ -156,8 +157,9 @@ class V3ErrorCode final {
     V3ErrorCode()
         : m_e{EC_MIN} {}
     // cppcheck-suppress noExplicitConstructor
-    constexpr V3ErrorCode(en _e)
-        : m_e{_e} {}
+    constexpr V3ErrorCode(en _e) VL_PURE : m_e{_e} {}
```

Comment:
FYI: seems like the CI compiles Verilator using  C++ version newer than 11, where this assumption is not true. Maybe it would be good to test with C++11?
For an example of non-pure constexpr method (invalid in C++11, valid since C++14) see the code two lines below:
```c++
constexpr void operator=(const V3ErrorCode& rhs) VL_PURE { m_e = rhs.m_e; }
```

Reply:
As far I know, currently libclang doesn't provide a way to check if function is ``contexpr``. We could check tokens, but it doesn't follow macros, typedefs etc. 

(Some old answer from stackoverflow: https://stackoverflow.com/questions/60627345/python-libclang-how-to-distinguish-const-vs-constexpr-types)